### PR TITLE
Add m4b mime type

### PR DESCRIPTION
**Pull request by vorte**
Currently, there is no mime type support for `.m4b` files (audiobooks). 

This PR maps the m4b extension to the `audio/mp4` mime type - same as m4a files.

### CODE REVIEW

**Comment by vorte** [comment]
@kevmoo @devoncarew could you have a look when you get a chance, please?

File: lib/src/default_extension_map.dart [hunk 1]
```diff
@@ -419,6 +419,7 @@ const Map<String, String> defaultExtensionMap = <String, String>{
   'm3u8': 'application/vnd.apple.mpegurl',
   // Source: https://www.rfc-editor.org/rfc/rfc4337#page-3
   'm4a': 'audio/mp4',
+  'm4b': 'audio/mp4',
```
Comment (kevmoo) [inline lib/src/default_extension_map.dart:422]: like above, please add a comment referencing the corresponding RFC, etc

**Review by kevmoo — CHANGES_REQUESTED** [review]
We'll also want a changelog entry

File: lib/src/default_extension_map.dart (hunk 1 above)
Comment (vorte) [reply lib/src/default_extension_map.dart:422]: Both .m4a and .m4b files are MP4 files which include audio only. The only difference is metadata - m4b supports bookmarks, which are needed for audiobooks. 

So the same RFC applies to both file extensions. I updated the link as the old one doesn't lead anywhere.

**Comment by vorte** [comment]
> We'll also want a changelog entry

Added 👍

**Comment by kevmoo** [comment]
@natebosch ?

File: .gitignore [hunk 2]
```diff
@@ -3,3 +3,4 @@
 .pub/
 packages
 pubspec.lock
+build/
```
Comment (natebosch) [inline .gitignore:6]: I don't know why we'd have a build directory in this repo... Is this an artifact of a local tool you are using? Consider adding this to a global ignore if so.

**Review by natebosch — APPROVED** [review]

File: .gitignore (hunk 2 above)
Comment (vorte) [reply .gitignore]: Hmm, when I ran the unit tests, they seemed to generate a build folder, so I added to `.gitignore`. I think I ran them within VS Code, so maybe that's why. 

I'm happy with removing that change, though.

**Review by kevmoo — APPROVED** [review]

**Pull request merged**